src/main/java/net/minecraftforge/registries/RegistryBuilder.java

```diff
@@ -230,6 +231,13 @@ public RegistryBuilder<T> hasTags()
         return this;
     }
 
+    public RegistryBuilder<T> isDataPackRegistry()
+    {
+        this.hasWrapper();// A Wrapper is required for data pack registries
```

Comment:
Wrapper shouldn't be capitalized, and there should be a space after the semicolon and before the comment start.